bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/items/ItemUIRegistryImpl.java

```diff
@@ -173,6 +174,9 @@ public Widget getWidget(String itemName) {
         for (ItemUIProvider provider : itemUIProviders) {
             Widget currentWidget = provider.getWidget(itemName);
             if (currentWidget != null) {
+                if (currentWidget instanceof Default) {
```

Comment:
Hm, I am a bit confused now. Actually, there isn't really a "fallback" - if it is NOT Default, we want the widget. If it IS Default and cannot be resolved, we want to have null (and not the Default widget) - so is it valid to talk about a fallback here?